ejb3/src/main/java/org/jboss/as/ejb3/subsystem/FilterSpecClassResolverFilter.java

```diff
@@ -84,8 +84,7 @@ final class FilterSpecClassResolverFilter implements Function<String, Boolean> {
                     + "!org.codehaus.groovy.runtime.ConvertedClosure;"
                     + "!org.codehaus.groovy.runtime.MethodClosure;"
                     + "!org.springframework.beans.factory.ObjectFactory;"
-                    + "!com.sun.org.apache.xalan.internal.xsltc.trax.TemplatesImpl;"
-                    + "!org.apache.xalan.xsltc.trax.TemplatesImpl";
```

Comment:
This should be kept. It does no harm and if somehow this class ended up on the classpath (e.g. in a deployment), removing it would introduce a vulnerability.  IIRC I created this list based on a set of known problems and I did not limit it to classes shipped by WildFly. I'm pretty sure some of the classes shown in the diff view here are not shipped by WildFly.

Reply:
Fixed.